src/components/Skeleton/index.tsx

```diff
@@ -39,28 +38,37 @@ export interface SkeletonProps extends StandardProps {
   isPulsing?: boolean;
 }
 
-const defaultBaseColor = whiteSmoke;
-const defaultHighlightColor = whiterSmoke;
+const defaultBaseColor = themed(({ theme }) => theme.ui3);
+const defaultHighlightColor = themed(({ theme }) => theme.ui2);
 
 const shine = keyframes`
   from {
-    background-position: -100px;
+    background-position: -200px;
   }
   to {
-    background-position: 110%;
+    background-position: 130%;
   }
 `;
 
+const Span = styled.span`
+  background-color: ${defaultBaseColor};
+  background-image: linear-gradient(90deg, ${defaultBaseColor}, ${defaultHighlightColor}, ${defaultBaseColor});
+  background-size: 60px 100%;
+  background-repeat: no-repeat;
+  display: inline-block;
+  line-height: 1;
+`;
+
 export const Skeleton: React.FC<SkeletonProps> = props => {
   const { count = 1, duration = 1.2, width = '100%', height = '100%', isCircle, isText, isPulsing = true } = props;
   const skeletons = [];
 
   for (let i = 0; i < count; i++) {
-    const extraStyle = css`
+    const StyledSpan = styled(Span)`
       animation: ${shine} ${duration}s infinite linear ${!isPulsing ? 'paused' : 'running'};
 
       ${count > 1 && isText
-        ? { width: `${Math.floor(Math.random() * (100 - 80 + 1) + 80)}%` }
+        ? { width: `${Math.floor(Math.random() * 21 + 80)}%` } // random number between 80 and 100
```

Comment:
I would create the function and use it here like `randInt(80, 100)`, which is more clear than just a comment. 

```ts
const randInt = (min: number, max: number) => 
	Math.floor(Math.random() * (max - min + 1)) + min;